CMakeLists.txt

```diff
@@ -9,8 +9,8 @@
 # Distributed under the Boost Software License, Version 1.0. (See accompanying
 # file LICENSE_1_0.txt or copy at http://www.boost.org/LICENSE_1_0.txt)
 
-# We require at least CMake V2.8.12
-cmake_minimum_required(VERSION 2.8.12 FATAL_ERROR)
+# We require at least CMake V3.0.2
+cmake_minimum_required(VERSION 3.0.2 FATAL_ERROR)
 
 # explicitly set certain policies
 cmake_policy(VERSION 2.8.12)
```

Comment:
I'm not saying this _has_ to be changed. I'm asking whether this is necessary... To answer this you'd need to understand what the `cmake_policy(<version>)` macro actually does.